package.json

```diff
@@ -82,12 +83,6 @@
         "coverage:unit": "jest --colors --config jest.config.js --coverage --testPathIgnorePatterns integration_test.js",
         "coverage:integration": "jest --colors --config jest.config.js --coverage integration_test.js",
         "coverage": "jest --colors --config jest.config.js --coverage",
-        "pretty-quick": "pretty-quick --staged",
         "lint": "flow && eslint --report-unused-disable-directives --config .eslintrc.json '{src,bin,__{tests,mocks}__}/**/*.js'"
-    },
-    "pre-commit": [
-        "pretty-quick",
-        "lint",
-        "test"
-    ]
```

Comment:
We have github actions now, we don't really need this overhead. Also, the linting was linting files that were not staged, which just got in the way.